CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/AttributeEditorPanel.java

```diff
@@ -868,9 +863,7 @@ private void modifyAttributeAction(final AttributeData attr) {
     }
 
     private void updateTimeZoneAction(final AttributeData attr) {
-        final EditOperation editOperation = (zoneId) -> {
-            PluginExecution.withPlugin(new UpdateTimeZonePlugin((ZoneId) zoneId, attr)).executeLater(GraphManager.getDefault().getActiveGraph());
-        };
+        final EditOperation editOperation = zoneId -> PluginExecution.withPlugin(new UpdateTimeZonePlugin((ZoneId) zoneId, attr)).executeLater(GraphManager.getDefault().getActiveGraph());
```

Comment:
Ditto here (although the line below is also stupidly long).

Reply:
Done